### ISSUE TITLE
test failure in travis

### ISSUE THREAD

**Issue by skoudoro** [nipy/dipy#1698]
## Description

Something to investigate. I suppose it is due to a new package release (Numpy or scipy). Below the error:

```
======================================================================
ERROR: dipy.core.tests.test_sphere.test_interp_rbf
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/dipy/core/tests/test_sphere.py", line 375, in test_interp_rbf
    interp_data_a = interp_rbf(data, s0, s1, norm="angle")
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/dipy/core/sphere.py", line 572, in interp_rbf
    **kwargs)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/scipy/interpolate/rbf.py", line 241, in __init__
    self.nodes = linalg.solve(self.A, self.di)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/scipy/interpolate/rbf.py", line 247, in A
    r = squareform(pdist(self.xi.T, self.norm))  # Pairwise norm
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/scipy/spatial/distance.py", line 2030, in pdist
    dm[k] = metric(X[i], X[j], **kwargs)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/dipy/core/sphere.py", line 544, in angle
    xx[np.isnan(xx)] = 0
TypeError: 'numpy.float64' object does not support item assignment
----------------------------------------------------------------------
```

**Comment by arokem** [comment]
@jhlegarreta : I just submitted a PR with a potential fix in #1708, so you'll probably want to rebase all of your PRs (what a 🎄gift!) once that's resolved.

**Comment by jhlegarreta** [comment]
@arokem Sure. Will do once it gets merged. Thanks for the fix and the note.
